packages/cypher-builder/src/clauses/With.test.ts

```diff
@@ -71,7 +71,7 @@ describe("CypherBuilder With", () => {
         });
 
         test("With expression aliased", () => {
-            const expr = Cypher.plus(new Cypher.Param("The "), new Cypher.Param("Matrix"));
+            const expr = Cypher.append(new Cypher.Param("The "), new Cypher.Param("Matrix"));
```

Comment:
Based on the comment above the export, do we want to consider exporting these functions under their namespace?

```suggestion
            const expr = Cypher.string.concat(new Cypher.Param("The "), new Cypher.Param("Matrix"));
```

This would also apply to math operators:

```ts
Cypher.math.multiply(literal1, literal2);
```

Reply:
I was thinking something the same, but I was not sure what should be exported under a different namespace and what to export from the root namespace!

Reply:
@angrykoala, any thoughts on the above?